app/src/main/java/com/randomappsinc/studentpicker/importdata/ImportFromFileActivity.java

```diff
@@ -54,7 +58,7 @@ public void onConfigurationChanged(@NonNull Configuration newConfig) {
         bannerAdManager.onOrientationChanged();
     }
 
-    private void extractNameListInfo() {
+    private void extractNameListInfoFromTextFile() {
```

Comment:
I was lazy and inlined all the file parsing logic within the activity. I think as we build this out, we should create a FileParsingManager class. We pass it the file to parse and it parses on the background thread. When it's done, it lets the activity know what it parsed via a callback.